onnxruntime/core/providers/nnapi/nnapi_builtin/builders/helper.cc

```diff
@@ -493,8 +493,11 @@ bool IsNodeSupportedInGroup(const NodeUnit& node_unit, const GraphViewer& graph_
   if (!IsNodeSupported(node_unit, graph_viewer, params))
     return false;
 
-  // TODO, ignore this step if the node_unit is qdq node_unit
   // We also want to check if the node is supported as an internal quantized node
+  // First, ignore QDQ NodeUnit which is not internal quantized node
+  if (node_unit.UnitType() == NodeUnit::Type::QDQGroup)
+    return true;
+
```

Comment:
Are these handled by the call to IsNodeSupported and we just want to ignore them for the remainder of this method?

Reply:
Modified it a little bit to make it clearer, the code after this does not really apply to QDQ node groups